### DISCUSSION TITLE
如何解决 "Saving to disk"问题

### DISCUSSION THREAD

**Discussion by Lawyer-tan** [5eagull/RemNote-Chinese-chat-group#37] [你问我答/互相帮助]
<img width="311" alt="2021-07-12 at 上午9 05" src="https://user-images.githubusercontent.com/87259440/125216323-6cc7e480-e2f0-11eb-805d-1e4a57351b65.png">

最近保存常出现 Saving to disk，经常要等 1 个小时左右才能恢复正常。
大家有遇到这种情况吗？是什么原因导致的。

备注： MAC  10.13系统，备份数据大概 6M 左右。

**Comment by lovedebug** [comment]
有没有升级版本？ 如果是在线库和离线版本建议卸载重装，浏览器的话清理掉所有 cookie 重新登陆
